### PR TITLE
fix wording for VPC peering steps

### LINKED ISSUE #2777: Contradicting VPC peering instructions
The instructions here for accepting the peering request seem wrong. I thought the idea was that we had the role then the customer would not have to accept it themselves

 https://github.com/astronomer/docs/blob/05759d7c3227b0a357f40a86a032efbbaefd38a9/astro/connect-aws.md?plain=1#L98

### PR DESCRIPTION
**Pull request by afastronomer**
Resolves #2777

### CODE REVIEW

File: astro/connect-aws.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ To create a VPC peering connection between an Astro VPC and an AWS VPC, you must
     
     Astronomer support will initiate a peering request and create the routing table entries in the Astro VPC.
 
-6. Wait for Astronomer support to send you the Astro VPC CIDR. Then, the owner of the target VPC needs to [accept the peering request](https://docs.aws.amazon.com/vpc/latest/peering/accept-vpc-peering-connection.html) and [create the routing table entries](https://docs.aws.amazon.com/vpc/latest/userguide/WorkWithRouteTables.html#AddRemoveRoutes) in the target VPC.
+6. Wait for Astronomer support to send you the Astro VPC CIDR and VPC Peering ID. Then, the owner of the accepter VPC needs to [create back routes entries](https://docs.aws.amazon.com/vpc/latest/userguide/WorkWithRouteTables.html#AddRemoveRoutes) in accepter VPC, with Astro VPC CIDR as Destination and provided VPC Peering ID as Target.
```
Comment (lzdanski) [inline astro/connect-aws.md:98]:
```suggestion
6. Wait for Astronomer support to send you the Astro VPC CIDR and VPC Peering ID. Then, the owner of the accepter VPC needs to [create back routes entries](https://docs.aws.amazon.com/vpc/latest/userguide/WorkWithRouteTables.html#AddRemoveRoutes) in the accepter VPC, with the Astro VPC CIDR as the **Destination** and the provided VPC Peering ID as the **Target**.
```

Comment (lzdanski) [reply astro/connect-aws.md:98]: @afastronomer or @stuart23 Making an assumption that the Destination and the Target are buttons or UI entry fields. If they're parameters or values defined in `code`, then they'll need to be code formatted instead. They do need to have a decorator in addition to capitalization. Can you confirm that these changes are accurate?

**Review by lzdanski — COMMENTED** [review]
Question about some wording and text formatting for style.

File: astro/connect-aws.md (hunk 1 above)
Comment (afastronomer) [reply astro/connect-aws.md:98]: Those are column names. Now it looks better.

**Review by jwitz — CHANGES_REQUESTED** [review]
Thanks for this update @afastronomer , just one question about the phrase "accepter VPC"!

**Review by jwitz — APPROVED** [review]
Confirmed with @afastronomer on Slack that "external" works here!

**Pull request merged**